client/sections.js

```diff
@@ -9,7 +9,7 @@ const sections = [
 	{
 		name: 'sites',
 		paths: [ '/sites' ],
-		module: 'calypso/my-sites',
+		module: 'calypso/sites-dashboard',
```

Comment:
I'm not sure, but I think our dashboard still might be better off as its own section. By taking over this section it means that we're going to inherit all styles that have been defined for `body.is-group-sites` and `body.is-section-sites`. And vice versa, all those existing sections are going to inherit the styles we define in `body.is-group-sites`.

If you look at the other sections with `group: 'sites'` in this file you get a feeling of the sort of views that are in that group, and the new dashboard feels distinct from them.